src/less/components/file/index.less

```diff
@@ -1,8 +1,8 @@
-@import (reference) 'components/box/index';
+//@import (reference) 'components/box/index';
 @import './hx-file-control';
```

Comment:
Can we remove this file altogether (`src/less/components/file/index.less`)? It doesn't look like it's loaded by LESS any more.

Reply:
Changes have been done, please review.

Note : for solving this i have commented "@import (reference) 'components/file/index';" in file-input-spec.less file. Anyhow this file will be deleted when #PR650 is merged.